src/test/java/com/jcabi/github/mock/MkCommentTest.java

```diff
@@ -90,6 +96,57 @@ public void canCompareInstances() throws Exception {
     }
 
     /**
+     * MkComment should store all its data properly.
+     * We should get the proper data back when accessing its properties.
+     * @throws Exception when a problem occurs.
+     */
+    @Test
+    public void dataStoredProperly() throws Exception {
+        final long before = this.now();
+        final Comment comment = this.comment();
+        final long after = this.now();
+        MatcherAssert.assertThat(
+            comment.number(),
+            Matchers.greaterThan(0)
+        );
+        final Comment.Smart smart = new Comment.Smart(comment);
+        MatcherAssert.assertThat(
+            smart.issue().number(),
+            Matchers.greaterThan(0)
+        );
+        MatcherAssert.assertThat(
+            smart.author().login(),
+            Matchers.equalTo("jeff")
+        );
+        MatcherAssert.assertThat(
+            smart.body(),
+            Matchers.equalTo(COMMENT_TEXT)
+        );
+        final String domain = "https://api.jcabi-github.invalid";
```

Comment:
@longtimeago The string hits the line-length limit or "don't concat constant strings" qulice checks otherwise. Can you suggest an alternative?